test_util/convertECL.cpp

```diff
@@ -166,6 +260,35 @@ int main(int argc, char **argv) {
     std::string extension = filename.substr(p,l-p);
     std::string resFile;
 
+    if (to_grdecl) {
+    
+        std::ofstream ofileH;
+        open_grdecl_output(output_fname, filename, ofileH);
+
+        auto array_list = file1.getList();
+
+        for (size_t n = 0; n < array_list.size(); n ++) {
+            std::string name = std::get<0>(array_list[n]);
+            auto arr_type = std::get<1>(array_list[n]);
+
+            if (arr_type == Opm::EclIO::REAL) {
+                auto  data = file1.get<float>(n);
+                writeGrdeclData(ofileH, name, data);
+            } else if (arr_type == Opm::EclIO::INTE) {
+                auto  data = file1.get<int>(n);
+                writeGrdeclData(ofileH, name, data);
+            } else {
+                std::cout << "unknown data type for array " << name << std::endl;
+                exit(1);
+            }
+        }
+
+        ofileH.close();
```

Comment:
This is fine, of course, but the `ofstream` object will close the associate stream object in its destructor so you can dispense with the explicit `.close()` call here if you want to.